apps/web/src/views/V3Info/data/token/priceData.ts

```diff
@@ -347,6 +335,41 @@ export async function fetchPairPriceChartTokenData(
       close: string
     }[] = []
 
+    const maxQueryPrice = isDay
+      ? max(
```

Comment:
may I ask why are we moving the min/max calculation from subgraph to FE?

Reply:
umm should we switch to thegraph for this?

Reply:
https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/chains/src/subgraphs.ts